editor/connections_dialog.cpp

```diff
@@ -1491,10 +1491,14 @@ void ConnectionsDock::update_tree() {
 				for (const MethodInfo &signal : base_signals) {
 					base_signal_names.insert(signal.name);
 				}
-				for (List<MethodInfo>::Element *F = class_signals.front(); F; F = F->next()) {
+				List<MethodInfo>::Element *F = class_signals.front();
+				List<MethodInfo>::Element *next_F;
+				while (F) {
+					next_F = F->next();
 					if (base_signal_names.has(F->get().name)) {
 						class_signals.erase(F);
 					}
+					F = next_F;
```

Comment:
```suggestion
				for (List<MethodInfo>::Element *F = class_signals.front(); F;) {
					List<MethodInfo>::Element *N = F->next();
					if (base_signal_names.has(F->get().name)) {
						class_signals.erase(F);
					}
					F = N;
```

For consistency with multiple other instances of the same code.

Reply:
Thanks!